plugins/DevicePlugins/DevicePlugins.php

```diff
@@ -45,11 +44,4 @@ public function addMetricTranslations(&$translations)
 
         $translations = array_merge($translations, $metrics);
     }
-
-    public function renameUserSettingsModuleAndAction(&$module, &$action)
-    {
-        if ($module == 'UserSettings' && $action == 'getPlugin') {
```

Comment:
@sgiehl why removing this as well, as it would break Embed widgets and is not needed to deprecate IMHO